libs/cardano-ledger-api/cardano-ledger-api.cabal

```
@@ -55,8 +55,8 @@ library
         aeson >=2.2,
         bytestring,
         cardano-ledger-allegra ^>=1.6,
-        cardano-ledger-alonzo >=1.9 && <1.11,
-        cardano-ledger-babbage ^>=1.9,
+        cardano-ledger-alonzo >=1.9 && <1.12,
+        cardano-ledger-babbage ^>=1.10,
```

There is no breakage resulted in `cardano-ledger-api` due to this chnage, so we can have more relaxed bounds.
```suggestion
        cardano-ledger-babbage >=1.9 && <=1.11,
```